src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -37,10 +37,44 @@
 #include "application_manager/application_manager.h"
 #include "application_manager/message_helper.h"
 #include "smart_objects/smart_object.h"
+
 namespace application_manager {
 
 namespace commands {
 
+std::string MergeInfos(const std::string& first, const std::string& second) {
+  return first + ((!first.empty() && !second.empty()) ? ", " : "") + second;
+}
+
+std::string MergeInfos(const std::string& first,
+                       const std::string& second,
+                       const std::string& third) {
+  std::string result  = MergeInfos(first, second);
+  return MergeInfos(result, third);
+}
+
+void CreateInfoForUnsupportedResult(std::string& info,
```

Comment:
It's preferable to use return value instead of output parameter.
It's simpler for understanding and allows to make return value `const`.